test/spec/modules/stroeerCoreBidAdapter_spec.js

```diff
@@ -823,13 +860,13 @@ describe('stroeerCore bid adapter', function() {
         });
 
         it('should handle no kvg config', () => {
-          getConfigStub.withArgs('kvg').returns(undefined);
+          getConfigStub.withArgs('kvg').returns({});
```

Comment:
May want to leave it as returning `undefined`.  That was the initial intention. Because a random config returns undefined (e.g. `pbjs.getConfig("blah");` will return undefined).

But testing for `{}` is valid because the wrapper will do this. I reckon just duplicate the test and call it 'should handle empty kvg config'.